src/lib/Components/Inbox/Conversations/Message.tsx

```diff
@@ -125,7 +125,7 @@ export class Message extends React.Component<Props, any> {
           {this.renderAttachmentPreviews(message.attachments)}
 
           <BodyText disabled={!isSent}>
-            {message.raw_text.split("\n\nAbout")[0]}
+            {message.body.split("\n\nAbout")[0]}
```

Comment:
Unrelated to this PR, but this will match lots of other messages (you can't start a paragraph with the word about now, right?)... maybe it should be more conservative?